### PR TITLE
Create ship

### PR DESCRIPTION
**Pull request by Rahul2325**
Implemented createShip Func

### CODE REVIEW

File: battleship.py [hunk 1]
```diff
@@ -62,7 +62,13 @@ def mousePressed(data, event, board):
...
 '''
 def emptyGrid(rows, cols):
-    return
+    grid=[]
+    for i in range(rows):
+        l1=[]
+        for j in range(cols):
+            l1.append(EMPTY_UNCLICKED)
+        grid.append(l1)
+    return grid
```
Comment (ChandaSukesh) [inline battleship.py:72]: Maintain a proper naming conversions.

**Review by ChandaSukesh — COMMENTED** [review]
Good.

File: battleship.py (hunk 1 above)
Comment (Rahul2325) [reply battleship.py:72]: I have added a comment for l1 in the file but its not getting updated in GitHub

**Review by LaxmiNarayanaMurthyVemuri — APPROVED** [review]
Good job @Rahul2325

**Pull request merged**